tests/pytests/functional/modules/test_win_useradd.py

```diff
@@ -0,0 +1,343 @@
+import pytest
+from saltfactories.utils import random_string
+
+import salt.modules.cmdmod
+import salt.modules.win_useradd as user
+from salt.exceptions import CommandExecutionError
+
+pytestmark = [
+    pytest.mark.destructive_test,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.windows_whitelisted,
+]
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {user: {"__salt__": {"cmd.run_all": salt.modules.cmdmod.run_all}}}
+
+
```

Comment:
```suggestion
```
You don't need this anymore

Reply:
OK, removed it and the direct module imports. Created a user fixture that returns `modules.user`.